src/frontegg-mock/src/lib.rs

```diff
@@ -189,20 +144,259 @@ impl FronteggMockServer {
             })
             .unwrap();
     }
+
+    pub fn auth_api_token_url(&self) -> String {
+        format!("{}{}", &self.base_url, AUTH_API_TOKEN_PATH)
+    }
+}
+
+fn decode_access_token(
+    context: &Context,
+    token: &str,
+) -> Result<TokenData<Claims>, jsonwebtoken::errors::Error> {
+    jsonwebtoken::decode(
+        token,
+        &context.decoding_key,
+        &jsonwebtoken::Validation::new(jsonwebtoken::Algorithm::RS256),
+    )
+}
+
+fn generate_access_token(
+    context: &Context,
+    email: String,
+    tenant_id: Uuid,
+    roles: Vec<String>,
+) -> String {
+    let mut permissions = Vec::new();
+    if roles
+        .iter()
+        .any(|r| r.as_str() == "MaterializePlatformAdmin")
+    {
+        permissions.push("materialize.environment.write".to_owned());
+        permissions.push("materialize.invoice.read".to_owned());
+    }
+    if roles.iter().any(|r| r.as_str() == "MaterializePlatform") {
```

Comment:
Is it possible to make this and `MaterializePlatformAdmin` passed in values? Then mzcompose can pass them through.

Reply:
Sure, I can do that. I'll do it in a follow up PR, though.